> Did
> 
> ```
> rm -r node_modules
> yarn
> ```
> 
> and all packages including snappy build fine +1

ditto! :checkered_flag: 

But if I remove yarn.lock (+ rm -rf node_modules) and install/build again: this happens: might not be related to this
```typescript
bash-5.1# yarn build
yarn run v1.22.15
$ yarn clean && lerna run build
$ rm -rf ./packages/*/lib ./packages/*/*.tsbuildinfo
lerna notice cli v4.0.0
lerna info Executing command in 15 packages: "yarn run build"
@chainsafe/lodestar-params: $ tsc -p tsconfig.build.json
@chainsafe/lodestar-utils: $ tsc -p tsconfig.build.json
@chainsafe/lodestar-params: ../../node_modules/@types/tar/index.d.ts:239:36 - error TS2709: Cannot use namespace 'MiniPass' as a type.
@chainsafe/lodestar-params: 239 export interface ReadEntry extends MiniPass, HeaderProperties {
@chainsafe/lodestar-params:                                        ~~~~~~~~
@chainsafe/lodestar-params: Found 1 error in ../../node_modules/@types/tar/index.d.ts:239
@chainsafe/lodestar-params: error Command failed with exit code 2.
@chainsafe/lodestar-params: info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
lerna ERR! yarn run build exited 2 in '@chainsafe/lodestar-params'
lerna WARN complete Waiting for 1 child process to exit. CTRL-C to exit immediately.
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

--- a/package.json
+++ b/package.json
@@ -52,6 +52,7 @@
     "eslint-plugin-prettier": "^4.0.0",
     "lerna": "^4.0.0",
     "mocha": "^9.2.2",
+    "node-gyp": "^9.0.0",
     "npm-run-all": "^4.1.5",
     "nyc": "^15.1.0",
     "prettier": "^2.0.5",

